views/templates/test_runner.tpl

```diff
@@ -1,6 +1,14 @@
 <?php
 use oat\tao\helpers\Template;
 use oat\tao\helpers\Layout;
+
+$contentPanelSetup = '';
+if (get_data('review_screen')) {
+    $contentPanelSetup .= ' data-review-screen="1"';
+}
+if (get_data('review_region')) {
+    $contentPanelSetup .= ' data-review-region="' . get_data('review_region') . '"';
+}
```

Comment:
There shouldn't be any construction in the template expect var,  if/then/else and for loops,